### ISSUE TITLE
Is possible graduate the level of blur effect?

### ISSUE THREAD

**Issue by frodfigu** [adrianopaladini/Live-blurView#6]
Most cases is so high.

Thank you!

**Comment by mark22** [comment]
Yes I would love this also.  Ive been using this module all over my app and its amazing, I love it.  I would just love the amount of blur to be less sometimes.  If we could control it would be the best

**Comment by KittenYang** [comment]
Take look at UIViewPropertyAnimation in iOS10.
